pkg/security/apiserver/admission/sccadmission/admission.go

```diff
@@ -133,6 +133,9 @@ func (c *constraint) computeSecurityContext(a admission.Attributes, pod *coreapi
 	if err != nil {
 		return nil, "", nil, admission.NewForbidden(a, err)
 	}
+	if len(constraints) == 0 {
+		return nil, "", nil, admission.NewForbidden(a, fmt.Errorf("no SecurityContextConstraints found in namespace %s", a.GetNamespace()))
```

Comment:
wrong spot.  YOu can still have a message here about no matching constraints for namespace, but do a separate list above this.

Reply:
only list all if didn't find any in the ns?